Resources/views/Admin/Security/Resetting/checkEmail.html.twig

```diff
@@ -27,6 +27,11 @@ file that was distributed with this source code.
     <div class="login-box">
         <div class="login-logo">
             <a href="{{ path('sonata_admin_dashboard') }}">
+                {% if 'single_image' == adminPool.getOption('title_mode') or 'both' == adminPool.getOption('title_mode') %}
```

Comment:
`if adminPool.getOption('title_mode') in ['single_image', 'both']`

Reply:
done